### ISSUE TITLE
Running xrandr-invert-colors does nothing on my system

### ISSUE THREAD

**Issue by harsh-a1** [zoltanp/xrandr-invert-colors#25]
Hi
When I run the command it has no effect on the screen.There is no error message or anything. 
![Selected_2022-09-18_811x129](https://user-images.githubusercontent.com/1791655/190899938-89d5e1c9-0ca1-400b-86b4-8fef3c18b1bc.png)

Am using arch
```
Linux xxxxx 5.15.67-1-lts #1 SMP Thu, 08 Sep 2022 12:59:52 +0000 x86_64 GNU/Linux
```

My graphics driver 
```
VGA compatible controller: Intel Corporation Atom/Celeron/Pentium Processor x5-E8000/J3xxx/N3xxx Integrated Graphics Controller (rev 21) 
```

What could be wrong?

**Comment by wwalker** [comment]
You are probably seeing this (#20) kind of problem.  I have this same problem on all my machines now. (with multiple different graphics cards, intel, amd)  `xrandr-invert-colors` doesn't work on them any more.   Neither does `xcalib -i -a` which I used before `xrandr-invert-colors` (which I swtiched to so I could invert single displays in a multi display set).

I can no longer invert any of the screens on any of my machines.

Try `xcalib -i -a` and see if you can get it to work as a troubleshooting step.

**Comment by harsh-a1** [comment]
Yes, xcalib is also not working for me. I have also asked on the xcalib repo about the same few weeks back...[ https://github.com/OpenICC/xcalib/issues/12 ] and also here [ https://unix.stackexchange.com/q/688865/240484 ]

It's too bad this does not work anymore as I used this feature a lot...

**Comment by wwalker** [comment]
me too, I spend a few hours, every time something reminds me about it, to get it to work "somehow", **_\*ANY\*_** way.

I noticed that `redshift` works, but I think it only uses xrandr to change the gamma and the contrast, but since I no longer have any machine that inverting works on, I can't even see what good values in xcalib look like.
